_posts/2018-07-31-how-to-move-to-https.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title: "How to transition your website from HTTP to HTTPS"
+date: 2018-07-31
+author: Chris Russo
+tags:  marketing user-experience best-practices performance drupal drupal-planet
+summary: Now that you know your website needs to be served over HTTPS, we help you get there.
+featured_image: "/blog/https2.jpg"
+featured_image_alt: "Image endorsing move to HTTPS from HTTP"
+drupal_planet_summary: |
+  How to move from serving your site over HTTP to HTTPS. Spoiler alert: there are some easy ways to do this with various Drupal hosting options. A quick read for guidance.
+---
+
+This is part two of a two-part series.
+
+In [part one](/2018/06/26/why-you-need-https.html), we cogently convinced you that regardless of what your organization does and what functionality your website has, it is in your best interest to serve your website securely over HTTPS exclusively. Here we provide some guidance as to how to make the move.
+
+## How to transition to HTTPS
+
+To fully transition to HTTPS from HTTP means to serve your website HTML and assets exclusively over HTTPS. This requires a few basic things:
+
+- A digital certificate from a certificate authority (CA)
+- Proper installation of the certificate on your website's server
+- Ensuring all assets served from your website are served over HTTPS
+
+Let's break those down
+
+### Acquire a digital certificate
+
+As briefly discussed in [part one](/2018/06/26/why-you-need-https.html), to implement HTTPS for your website you must procure a digital certificate from a certificate authority. Just like domain name registrars lease domain names, CAs lease digital certificates for a set time period. Each certificate has a public and private component. The public component is freely shared and allows browsers to recognize that a "trusted" CA was the one to distribute the certificate and it is also used to for encrypt data transmitted from the browser. The private complement is only shared with the purchaser of the certificate, and can uniquely decrypt data encrypted by the public key. CAs use various methods through email or DNS to "prove" that the person who purchased the certificate is a rightful administrator of the domain for which they purchased it. Once you've received the private key associated with the certificate, you'll need to install it on your server. Annual certificate costs range from up to $1,000 to free. More on that in a moment.
+
```

Comment:
Split into 2 sentences. Also extra `for`.

"The public component is freely shared and allows browsers to recognize that a "trusted" CA was the one to distribute the certificate. It is also used to ~for~ encrypt data transmitted from the browser."

Reply:
Suggest - "Certificate costs range from free to up to $1,000 annually."